terraform/modules/ecs/ecs-tasks.tf

```diff
@@ -190,7 +190,7 @@ resource "aws_ecs_service" "sync" {
   name            = "${var.prefix}-sync"
   cluster         = var.ecs_cluster_name == "" ? module.ecs[0].cluster_arn : data.aws_ecs_cluster.cluster[0].arn
   task_definition = aws_ecs_task_definition.sync.arn
-  desired_count   = 1
+  desired_count   = 2
```

Comment:
I wonder if we should add a variable and default it to 2 so folks don't necessarily have to incur extra costs if they prefer savings over availability.